src/components/JournalRouter/index.jsx

```diff
@@ -33,7 +34,8 @@ class JournalRouter extends React.Component {
         <PrivateRouteContainer exact path="/:journalAboutId" component={JournalPageRedirectContainer} />
         <Route path="/:journalAboutId/about" component={JournalAboutPageContainer} />
         <PrivateRouteContainer path="/:journalAboutId/pages/:pageId" component={JournalPageContainer} />
-        <PrivateRouteContainer path="/:journalAboutId/preview/:previewId" component={JournalPreviewContainer} />
+        <PrivateRouteContainer path="/:journalAboutId/pagePreview/:previewId" component={JournalPreviewContainer} />
```

Comment:
With two routes, it might not be worth it right now, but if we end up doing the same for previewing index pages I would probably suggest having a `JournalPreviewRouter` that runs off of `:/journalAboutId/preview/` and has routes for `<type>/:previewId`. That way the `JournalPreviewRouter` can handle all of the preview fetching and routing. Again, definitely not a requirement now, but if we're planning on adding them, I'd keep a potential refactor of some routing and fetching logic in mind.

Reply:
thanks, I will make this refactor when I add the index page stuff in an upcoming PR